### DISCUSSION TITLE
Converting vdi to ova

### DISCUSSION THREAD

**Discussion by Dvalin21** [home-assistant/operating-system#3108] [Q&A] [answered]
I've installed HAOS on VM application on Synology NAS. In order to do so I have to take the virtualbox image and convert to a ova image. Now what I noticed is that when I convert it and run it on the Synology, it moves SOOO slow. Even when I make available 4 cores and 6g of ram it takes forever!! Would I have the same result if I took the .qcow2 image and convert it to ova? Can I even do that?  Thanks.

**Comment by sairon** [comment]
We also publish images in the OVA format, why not use it directly?

**Comment by Dvalin21** [comment]
When I go to the site, the only thing I see is the qcow2, vdi, and img
format. I guess I'm overlooking it. Can you share the link?

**Comment by Impact123** [comment, marked as answer]
Check `Assets` on the releases page: <https://github.com/home-assistant/operating-system/releases/latest>
![image](https://github.com/home-assistant/operating-system/assets/899193/b60f8dcf-5d1d-42a6-9587-08431747f24d)

**Reply by Dvalin21** [reply]
Wow, did I miss this. Thanks for the link!

**Reply by sairon** [reply]
For the record, the link is also in the documentation, you just need to view the "Install Home Assistant on other systems" page: https://www.home-assistant.io/installation/alternative

**Comment by Dvalin21** [comment]
> For the record, the link is also in the documentation, you just need to view the "Install Home Assistant on other systems" page: https://www.home-assistant.io/installation/alternative

Appreciate it. Ill keep all this noted for future reference.
